### DISCUSSION TITLE
easy way to install it on windows?

### DISCUSSION THREAD

**Discussion by carlmoss22** [leejet/stable-diffusion.cpp#381] [Q&A]
hi guys, is there a tuturial how to install it on a windows 10 computer? i downloaded the cudart-sd-bin-win-cu12-x64.zip but have no idea what to do with the dll files.

thx in advance!

**Comment by Green-Sky** [comment]
"cudart" stands for cuda-runtime, which is only the dlls needed for 
sd-master-xxxxx-bin-win-cuda12-x64.zip

**Reply by carlmoss22** [reply]
thx,but how to install it? i am no cuda expert :-(

**Reply by Green-Sky** [reply]
the "installation" is extracting both archives into the same directory
